.github/workflows/release.yml

```
@@ -0,0 +1,130 @@
+name: release
+on:
+  push:
+    tags:
+      - "v*"
```

```suggestion
      - "[0-9]+.[0-9]+.[0-9]+*"
```
@michaeljones do you mean this? I don't know if they support this syntax but feel free to commit the suggestion and merge

---

Can also remove the filter and it would build on all pushed tags I think

---

I think they do: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Sorry, I should have made it a suggestion to begin with.